lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -72,7 +72,10 @@ def term_results_id_ldpath
       # Return results id_predicate
       # @return [String] the configured predicate to use to extract the id from the results
       def term_results_id_predicate
-        Config.predicate_uri(term_results, :id_predicate)
+        return @pred_id unless @pred_id.blank?
+        @pred_id = Config.predicate_uri(term_results, :id_predicate)
```

Comment:
A config can specify that either a simple id (e.g. 'sh12345') or the full uri will be used to find a term.  If the config indicates a simple id will be passed in, then it needs to be able to find the subject URI that has the ID as the object value.  To do this, it does a query...  `s? <id_pred> '_ID_'`.  To execute this query it needs to know the `<id_pred>`.  

When using a predicate map, the predicate map tells you the `<id_pred>`.  When using an ldpath map, the id predicate needs to be created from the prefixes and the ldpath map.

When `term_results_id_predicate` is called, it either returns the predicate from the predicate_map or calls `pred_id_from_ldpath` to generate the id predicate.